purchase_order_line_sequence/views/report_purchaseorder.xml

```diff
@@ -5,12 +5,12 @@
         inherit_id="purchase.report_purchaseorder_document"
     >
         <xpath expr="//table/thead/tr/th[1]" position="before">
-            <th><strong>Sequence</strong></th>
+            <th><strong>Line Number</strong></th>
```

Comment:
Would you consider just making it `No.` to keep it short?

Reply:
Hi @yostashiro,
We have chosen this name for the field to follow the criteria used by Odoo in other standard fields, such as Serial Number. They do not employ abbreviations like No. or Nº for labels.

Reply:
I see. I thought this case could be different to keep a good balance between the label and the content - the content is usually only 1 or 2 digits.  But I'm OK if other people think otherwise.  Thank you very much for this improvement.